ci/docker/run.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+# Prep CI Enviroment
+if [[ "${CI}" = "true" ]]; then
+    set -exo pipefail
+fi
+
+# Receive and set variables.
+if [[ "${TRAVIS}" = "true" ]]; then
+    echo "Running build on Travis CI"
+    branch_name="dev"
+    git_repo="https://github.com/rchain/rchain"
+    docker_dst_repo="rchain/rnode:${TRAVIS_BRANCH}"
+elif [[ $1 && $2 && $3 ]]; then
+    echo "Running custom build"
+    branch_name=$1
+    git_repo=$2
+    docker_dst_repo="$3"
+else
+    echo "Invalid number of parameters."
+    echo "Example: sudo $0 <branch name> <repo url> <docker hub repo:tag>"
```

Comment:
I think this should be "Usage: ..." Also, remove `sudo` from the command. If people want to run this as root, that seems a bit crazy to me, but that's their call. We shouldn't tell them they have to.